docs/servers/server-software.md

```diff
@@ -66,6 +67,7 @@ Alongside the Vanilla BDS offering, many community projects exist, in a variety
 -   [Nukkit PetteriM1 Edition](https://github.com/PetteriM1/NukkitPetteriM1Edition)
 -   [PowerNukkitX](https://github.com/PowerNukkitX/PowerNukkitX)
 -   [Sculk](https://github.com/sculkmp/Sculk)
+-   [Blocky](https://github.com/JoseLuisHD/Blocky)
```

Comment:
Move this line underneath "Allay" so that alphabetical ordering is maintained.

Reply:
Thanks for the comment. I have already uploaded the requested changes.